### ISSUE TITLE
Command failed

### ISSUE THREAD

**Issue by AshHoe** [armin-pfaeffle/sass-autocompile#117]
<img width="978" alt="screen shot 2017-06-03 at 12 56 10 am" src="https://cloud.githubusercontent.com/assets/22785122/26736169/85d439f8-47f7-11e7-86c1-cfa21587e3ef.png">

**Comment by nrbaker02** [comment]
I have the same issue. Did you ever figure a resolution?

**Comment by nrbaker02** [comment]
@armin-pfaeffle, do you have any insight as to what could be causing this?

**Comment by nrbaker02** [comment]
I don't believe I installed node-sass globally as you clearly state in your readme. Uninstalled and reinstalled globally. Works great! 👍🏼
